gc3libs/config.py

```diff
@@ -401,6 +400,16 @@ def _parse(self, stream, filename=None):
                                 filename=filename
                             ))
 
+                for key in self.resource_key_value_matcher.keys():
```

Comment:
I would find the code more readable if the key/value matcher was unpacked like this::

    key_re, problem = self.resource_key_value_matcher[key]

then you can use `key_re` and `problem` in the code below, instead of `self.resouce_...[0]` and `...[1]`.